curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b23a413d28da927e087ca.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 657b23a413d28da927e087ca
+title: Task 52
+challengeType: 22
+dashedName: task-52
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+This task reviews how an adverb is often formed by adding `-ly` to an adjective. For example, `quick` (adjective) becomes `quickly` (adverb). Adverbs describe how, when, where, or to what extent something is done. However, not all adverbs follow this pattern. For instance, `most` is an adjective meaning the greatest in amount or degree, while `mostly` is an adverb meaning mainly or generally. 
```

Comment:
```suggestion
This task reviews how an adverb is often formed by adding `-ly` to an adjective.

```

Reply:
There's another block that teaches this. So I've changed this challenge for practice.